src/main/java/frc/robot/subsystems/LiftSubsystem.java

```diff
@@ -97,14 +104,15 @@ public double get() {
     }
 
     protected void restrictedMotorSet(double liftSpeed) {
+        if (armsPiston.get() == Value.kForward) {
+            liftSpeed = 0;
+        }
         if (liftSpeed > 0 && !upperLimit.get()) {
             liftSpeed = 0;
-
         }
         if (liftSpeed < 0 && !lowerLimit.get()) {
             liftSpeed = 0;
             heightEncoder.reset();
-
         }
         if (Math.abs(liftSpeed) <= 0.05) {
```

Comment:
We could get fancy here and use different values for up and down movement.

Up should be the value required to hold the lift in position
Down should be no limit, the deadband should be handled at the joystick.

Reply:
holding off for l8ter